tests/tensor_search/test_throttling.py

```diff
@@ -0,0 +1,76 @@
+import time
```

Comment:
We can leave this as a shell to fill in when we have time to get mocking set up properly

Reply:
definitely